indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -42,97 +47,177 @@ SidebarTrigger.propTypes = {
     active: PropTypes.bool.isRequired
 };
 
-function SidebarMenu({
-    isAdmin,
-    isAdminOverrideEnabled,
-    hasOwnedRooms,
-    gotoMyBookings,
-    gotoBookingsInMyRooms,
-    gotoMyRoomsList,
-    gotoMyBlockings,
-    gotoRBAdminArea,
-    toggleAdminOverride,
-    visible,
-    onClickOption,
-    hideOptions,
-}) {
-    const options = [
-        {
-            key: 'my_bookings',
-            icon: 'list alternate outline',
-            text: Translate.string('My Bookings'),
-            onClick: gotoMyBookings,
-        },
-        {
-            key: 'bookings_my_rooms',
-            icon: 'checkmark',
-            text: Translate.string('Bookings in My Rooms'),
-            onClick: gotoBookingsInMyRooms,
-            onlyIf: hasOwnedRooms
-        },
-        {
-            key: 'my_rooms',
-            icon: 'user',
-            text: Translate.string('List of My Rooms'),
-            onClick: gotoMyRoomsList,
-            onlyIf: hasOwnedRooms
-        },
-        {
-            key: 'my_blockings',
-            icon: 'window close outline',
-            text: Translate.string('My Blockings'),
-            onClick: gotoMyBlockings,
-            onlyIf: !hideOptions.myBlockings
-        },
-        {
-            key: 'isAdmin',
-            icon: 'cogs',
-            text: Translate.string('Administration'),
-            onClick: gotoRBAdminArea,
-            onlyIf: isAdmin
-        },
-        {
-            key: 'adminOverride',
-            icon: isAdminOverrideEnabled ? 'unlock' : 'lock',
-            text: Translate.string('Admin Override'),
-            iconColor: isAdminOverrideEnabled ? 'orange' : undefined,
-            active: isAdminOverrideEnabled,
-            onClick: toggleAdminOverride,
-            onlyIf: isAdmin,
-            tooltip: Translate.string('Admin Override gives you unrestricted access to all rooms and bookings.')
+class SidebarMenu extends React.Component {
```

Comment:
I don't think we need to convert this to a class component :)

When moving the URLs to the config store (see my previous comment) those are provided by redux, and the simple state used here right now would be a perfect use case for hooks otherwise!